email-client/src/main/java/uk/gov/hmcts/cmc/email/sendgrid/SendGridClient.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.hmcts.cmc.email.sendgrid;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Attachments;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import org.apache.http.HttpException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.cmc.email.EmailSendFailedException;
+
+import java.io.IOException;
+
+@Component
+public class SendGridClient {
+    private final SendGrid sendGrid;
+
+    @Autowired
+    public SendGridClient(
+        SendGridFactory factory,
+        @Value("${sendgrid.api-key}") String apiKey,
+        @Value("false") Boolean testing
```

Comment:
```suggestion
        @Value("${sendgrid.testing: false}") Boolean testing
```
So that we can override in configurations if needed.

Reply:
Meh. Sure.